### DISCUSSION TITLE
Errore listIssuedDocuments > DateTime::__construct(): Failed

### DISCUSSION THREAD

**Discussion by manuelzlp** [fattureincloud/api#371] [Q&A] [answered]
Buongiorno, ho il seguente errore nella chiamata IssuedDocumentsApi->listIssuedDocuments

Exception when calling IssuedDocumentsApi->listIssuedDocuments: DateTime::__construct(): Failed to parse time string (23062214282314126-000004) at position 7 (4): Double date specification

L'errore si presenta soltanto quando metto $fieldset = 'detailed'
In data 2023 06 22 avevamo fatto dei test, ma abbiamo cancellato tutte le fatture (non si trovano nemmeno da pannello FEIC) quindi adesso non saprei come correggere i dati e aggiustare il formato della data in questione

Grazie mille e buona giornata!

**Comment by valmoz** [comment]
Buongiorno @manuelzlp,
le chiederei gentilmente, se non già fatto, di scrivere al supporto clienti per verificare più in dettaglio il problema. Purtroppo ci serve il company id per capire cosa sta succedendo, che essendo un dato sensibile non può essere condiviso sulla community.

Le suggerisco di aggiungere il link a questa conversazione per velocizzare la gestione della richiesta di supporto.

Grazie
Mauro

**Reply by valmoz** [reply, marked as answer]
Buongiorno @manuelzlp è riuscito a risolvere con il supporto clienti?

Mauro
